crux-test/test/crux/query_test.clj

```diff
@@ -1421,7 +1421,7 @@
                                       :name "Ivan 3rd"}]
                                     [:crux.tx/cas
                                      {:crux.db/id :ivan
-                                      :name "Ivan 2nd"}
+                                      :name "Ivan 3rd"}
```

Comment:
this change to an old test shows that the CaS behaviour has changed when you've got two CaS operations to the same entity in the same tx